test/recipes/15-test_rsaoaep_data/plain_text

```diff
@@ -0,0 +1 @@
+Hello world
```

Comment:
Deliberately not including the final new line or not?

Reply:
Added